_pages/sponsors.md

```diff
@@ -25,7 +25,18 @@ Founded late in the 20th century (1997), the Kitchener Waterloo Software Quality
 
 <hr>
 
-## Why You Should Sponsor Us
+## Community Partners
+
+These organizations are doing really cool things! One of those cool things is helping spread the word about StarCon to their communities.
+
+Thanks so much to:
+
+- [Girl Geek Dinners Waterloo Region](https://www.meetup.com/Girl-Geek-Dinners-Waterloo-Region/?_cookie-check=bCWkq1imC0ofOyH-)
```

Comment:
just making sure you want the cookie-check here?

Reply:
heh no